static/scripts/javascript.js

```diff
@@ -372,6 +374,61 @@ var app = {
     });
   },
 
+  smooth_scroll: function() {
+    $(function() {
+      $('a[href*="#"]:not([href="#"])').click(function() {
+        if (
+          location.pathname.replace(/^\//, "") ===
+            this.pathname.replace(/^\//, "") &&
+          location.hostname === this.hostname
+        ) {
+          var target = $(this.hash);
+          target = target.length
+            ? target
+            : $("[name=" + this.hash.slice(1) + "]");
+          if (target.length) {
+            var scrollx = target.offset().top - 100;
+            $("html, body").animate(
+              {
+                scrollTop: scrollx
+              },
+              1000
+            );
+            target.focus(); // Setting focus
+            if (target.is(":focus")) {
+              // Checking if the target was focused
+              return false;
+            } else {
+              target.attr("tabindex", "-1"); // Adding tabindex for elements not focusable
+              target.focus(); // Setting focus
+            }
+            return false;
+          }
+        }
+      });
+    });
+  },
+
+  results_nav: function() {
+    window.onscroll = function() {
```

Comment:
It's better to do `window.addEventListener("scroll", function() { whatever });` because using `=` wipes out any other scroll events. I'm not sure but I think this will break the candidate page?

Reply:
it doesn't, but i'll change it.